.github/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ### New features
 
+* Adds the function `random_banded_interferometer` to generate unitary matrices with a given bandwidth. [#208](https://github.com/XanaduAI/thewalrus/pull/208)
+
```

Comment:
Is it worth mentioning `random_block_interferometer` also as a new feature, or is that not meant to be user-facing?

Reply:
Not too sure, seems a bit useless on its own.